examples/ansible/setup_libvirt_host.yml

```diff
@@ -1,129 +1,120 @@
 ---
-
-#
-# A example playbook showing how the setup an ALP system as a libvirt host.
-#
-
+# Ansible Playbook: Setup SUSE ALP Dolomite as a libvirt Host
+# Description: This Ansible playbook automates the setup of a SUSE ALP Dolomite host as a libvirt host.
+# The steps encompass installing necessary packages for the workload, ensuring system readiness through reboots if necessary,
+# fetching the required images for kvm-server and kvm-client from the specified registry, and installing tools for both kvm-server
+# and kvm-client. Subsequent tasks ensure that needed services are stopped, started, or enabled as per the requirements.
+# Documentation reference: [https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-kvm-with-podman]
+# Creating customized VMs using virt-scenario: [https://documentation.suse.com/alp/dolomite/html/alp-dolomite/concept-virt-scenario.html]
 
 - name: Setup ALP system as a libvirt host
   hosts: alphost
-  vars:
-    workload:
-      name: libvirtd
-      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/kvm:latest
-      labels:
-        install: install
-        uninstall: uninstall
-      required_pkgs:
+  become: true
+
+  tasks:
+    - name: Install required packages, if any, for workload
+      ansible.builtin.package:
+        name: "{{ item }}"
+        state: present
+      notify: Reboot
+      loop:
         - kernel-default
-        - "-kernel-default-base"
```

Comment:
FYI there is no harm leaving this directive in the list, as it ensures that the conflicting kernel is removed as part of installing the required kernel... It shouldn't be needed with the current ALP images, but would be useful if someone messes up in the furture ;-)